README.md

```diff
@@ -1219,7 +1219,7 @@ Available options:
 
 Some plugins can rely on this history, for instance:
  - [html-reporter](https://github.com/gemini-testing/html-reporter)
- - [hermione-profiler](https://github.com/gemini-testing/hermione-profiler)
+ - [hermione-profiler](https://github.com/gemini-testing/testplane-profiler)
```

Comment:
> hermione-profiler

Reply:
And so on…

Reply:
fixed:
plugin name `"hermione-..."` -> `"@testplane/..."`
plugin link `"hermione-..."` -> `"testplane-..."`, since we are still not in a monorepo